app/controllers/smart_answers_controller.rb

```diff
@@ -116,24 +116,12 @@ def set_expiry(duration = 30.minutes)
   end
 
   def setup_navigation_helpers_and_content_item
-    @content_item = Services.content_store.content_item("/" + params[:id]).to_hash
-
-    # The GOV.UK analytics component[1] automatically sets `govuk:analytics:organisations`
-    # if there's a `organisations` key in the links. This will be sent to Google
-    # Analytics At the moment we want to avoid setting this because it will flood
-    # the analytics reports with (unexpected) data. We are currently working on
-    # a solution to this conundrum[2].
-    #
-    # [1] http://govuk-component-guide.herokuapp.com/components/analytics_meta_tags
-    # [2] https://trello.com/c/DkR63grd
-    if @content_item["links"]
-      @content_item["links"].delete("organisations")
-    end
-
-    @navigation_helpers = GovukNavigationHelpers::NavigationHelper.new(@content_item)
-  rescue GdsApi::HTTPNotFound, GdsApi::HTTPGone
     @navigation_helpers = nil
```

Comment:
(You could maybe move this `nil` assignment closer to the `@content_item.present?` check? Using either the ternary operator or the return from the `if` block?)